### ISSUE TITLE
EMAGE: How many speakers are used in the training?

### ISSUE THREAD

**Issue by weihaosky** [PantoMatrix/PantoMatrix#143]
Thanks for this excellent work!
From the config `emage.yaml` or VQVAE configs `cnn_vqvae_upper_30.yaml `, there is only one speaker "speaker 2" used in the training.
Is the pre-trained model also trained on only one speaker?

If this is true, why don't you use all speakers?

**Comment by H-Liu1997** [comment]
Thanks for your positive feedback on emage, yes only speak 2 is used and
included in pretrained model.

But for the skeleton cnn based ae for evaluation we use all 30 speakers
data for training

**Comment by weihaosky** [comment]
> Thanks for your positive feedback on emage, yes only speak 2 is used and included in pretrained model. But for the skeleton cnn based ae for evaluation we use all 30 speakers data for training

So all the pretrained models of vqvae is trained on all 30 speakers, which is different to the configs, e.g., `cnn_vqvae_upper_30.yaml`, `cnn_vqvae_lower_30.yaml`, `cnn_vqvae_hands_30.yaml`, `cnn_vqvae_face_30.yaml`, `cnn_vqvae_lower_foot_30.yaml`.
Do I understand right?

**Comment by H-Liu1997** [comment]
> > Thanks for your positive feedback on emage, yes only speak 2 is used and included in pretrained model. But for the skeleton cnn based ae for evaluation we use all 30 speakers data for training
> 
> 
> 
> So all the pretrained models of vqvae is trained on all 30 speakers, which is different to the configs, e.g., `cnn_vqvae_upper_30.yaml`, `cnn_vqvae_lower_30.yaml`, `cnn_vqvae_hands_30.yaml`, `cnn_vqvae_face_30.yaml`, `cnn_vqvae_lower_foot_30.yaml`.
> 
> Do I understand right?

Yes!

**Comment by weihaosky** [comment]
> > So all the pretrained models of vqvae is trained on all 30 speakers, which is different to the configs, e.g., `cnn_vqvae_upper_30.yaml`, `cnn_vqvae_lower_30.yaml`, `cnn_vqvae_hands_30.yaml`, `cnn_vqvae_face_30.yaml`, `cnn_vqvae_lower_foot_30.yaml`.
> > Do I understand right?
> 
> Yes!

Thanks. May I ask:
1) why the vqvae configs only use one speak?
2) would the performance be better if I use all speakers for training the autio2motion transformer network?

**Comment by H-Liu1997** [comment]
Let me clarify 
1. AE for calculating FGD is trained on all data
2. The remains, 5VQVAE, EMAGE, are trained in one speaker

It's motivated by performance, training will more speakers will cover more styles if you change the speaker embedding during inference. but it will decrease the FGD.

**Comment by weihaosky** [comment]
What does "AE for calculating FGD" refer to?
Are the provided models in PantoMatrix/EMAGE/pretrained_vq/ (e.g., upper_vertex_1layer_710.bin) trained on one speaker or all speakers?

**Comment by H-Liu1997** [comment]
The pretained skeleton cnn based auto encoder for calculating the fgd.

No, these vqvae are trained in one speaker data

**Comment by weihaosky** [comment]
I see. Thanks for you reply!

**Comment by wubowen416** [comment]
@H-Liu1997 
How many speakers are included to compute the Beat Constancy (BC) in Table 8, Appendix B, for BEATX dataset?

I obtained a BC score of 0.7086 for speaker 2 only on the test data using the original 30 fps data. When interpolation is used, the score becomes 0.6961.

On the whole data, I obtained 0.4853 when not using interpolation.

This score in the paper is 0.6896. How can I reproduce it?
